treeherder/etl/buildbot.py

```diff
@@ -914,8 +914,10 @@
     "Unknown": "?",
 }
 
-# Match the job part number from buildernames such as "... mochitest-5"
-NUMBER_RE = re.compile(r".*-(\d+)$")
+# Match the job part number from buildernames such as "... mochitest-5" or
+# "l10n 2/3"
```

Comment:
We don't enforce a max line length, so this doesn't need to be wrapped (80 chars is quite 1990s :-)).